source/github-helpers/selectors.ts

```diff
@@ -136,6 +136,7 @@ export const newCommentField = [
 export const newCommentField_ = [] satisfies UrlMatch[];
 
 export const commitHashLinkInLists = [
+	'[data-testid="commit-row-browse-repo"]',
```

Comment:
We need a comment like the ones below. Where does this apply and how is it different? Which one does this deprecates? We should mark the old one with "TODO: remove in May 2025”

Reply:
I wonder how we know the date.

Reply:
We leave the old selectors in for ~6 months to attempt to support GitHub Enterprise. We only do this when it's easy to support it.